test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java

```diff
@@ -0,0 +1,224 @@
+    for (int i : h3Rows) {
+      hashes.get(row(i)).add(h3);
+    }
+
+    int[] h4Rows = new int[] {300, 301, 672, 998};
+    String h4 = writeNonContiguousData(fs, dir + "/f4.", aconf, h4Rows);
+    for (int i : h4Rows) {
+      hashes.get(row(i)).add(h4);
+    }
+
+    final LoadPlan loadPlan =
+        LoadPlan.builder().loadFileTo("f1.rf", RangeType.FILE, row(0), row(11))
+            .loadFileTo("f2.rf", RangeType.TABLE, row(10), row(11))
+            .loadFileTo("f2.rf", RangeType.FILE, row(199), row(200))
+            .loadFileTo("f2.rf", RangeType.TABLE, row(203), row(204))
+            .loadFileTo("f3.rf", RangeType.TABLE, row(12), row(13))
+            .loadFileTo("f3.rf", RangeType.TABLE, row(199), row(200))
+            .loadFileTo("f3.rf", RangeType.FILE, row(272), row(273))
+            .loadFileTo("f4.rf", RangeType.FILE, row(300), row(301))
+            .loadFileTo("f4.rf", RangeType.TABLE, row(671), row(672))
+            .loadFileTo("f4.rf", RangeType.TABLE, row(997), row(998)).build();
+
+    try (AccumuloClient c = Accumulo.newClient().from(getClientProps()).build()) {
+
+      c.tableOperations().create(tableName, ntc);
+      TableId tid = TableId.of(c.tableOperations().tableIdMap().get(tableName));
+
+      final SortedSet<Text> metadataSplits = new TreeSet<>();
+      Scanner s = c.createScanner(MetadataTable.NAME, Authorizations.EMPTY);
+      final String mdTablePrefix = tid.canonical() + ";";
+      s.forEach(e -> {
+        final String row = e.getKey().getRow().toString();
+        if (row.startsWith(mdTablePrefix)) {
+          metadataSplits.add(new Text(row + "\\x00"));
+        }
+      });
+      c.tableOperations().addSplits(MetadataTable.NAME, metadataSplits);
+
+      c.tableOperations().importDirectory(dir).to(tableName).plan(loadPlan).load();
+
+      verifyData(c, tableName, new int[] {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 199, 200, 204,
+          272, 273, 300, 301, 672, 998}, false);
+      verifyMetadata(c, tableName, hashes);
+    }
+  }
+
+  public static void verifyData(AccumuloClient client, String table, int[] expectedRows,
+      boolean setTime) throws Exception {
+    try (Scanner scanner = client.createScanner(table, Authorizations.EMPTY)) {
+
+      Iterator<Entry<Key,Value>> iter = scanner.iterator();
+
+      int count = 0;
+      while (iter.hasNext()) {
+        Entry<Key,Value> entry = iter.next();
+
+        String expectedRow = String.format("%04d", expectedRows[count]);
+
+        if (!entry.getKey().getRow().equals(new Text(expectedRow))) {
+          throw new Exception("unexpected row " + entry.getKey() + " " + expectedRow);
+        }
+
+        if (Integer.parseInt(entry.getValue().toString()) != expectedRows[count]) {
+          throw new Exception("unexpected value " + entry + " " + expectedRows[count]);
+        }
+
+        if (setTime) {
+          assertEquals(1L, entry.getKey().getTimestamp());
+        }
+
+        count++;
+      }
+      if (iter.hasNext()) {
```

Comment:
is this if statement ever reached?

Reply:
again, left over from old code. I can remove, for now it will never get in here.

Reply:
I copied this from BulkNewIT, then modified the method, forgot to remove that block.